python/nav/portadmin/handlers.py

```diff
@@ -278,6 +290,24 @@ def is_configurable(self) -> bool:
             return False
         return True
 
+    def get_poe_state_options(self) -> Tuple[PoeState, ...]:
+        """Returns the available options for enabling/disabling PoE on this netbox"""
+        raise NotImplementedError
+
+    def set_poe_state(self, interface: manage.Interface, state: PoeState):
+        """Set state for enabling/disabling PoE on this interface.
+        Available options should be retrieved using `get_poe_state_options`
+        """
+        raise NotImplementedError
+
+    def get_poe_state(self, interface: manage.Interface) -> PoeState:
+        """Returns current PoE state of this interface"""
+        raise NotImplementedError
+
```

Comment:
When loading portadmin, youre gonna want to call get_poe_state for all  poe enabled ports. Doing it one by one is not very efficient. Perhaps adding another function for getting ALL poe states would make sense